sources/packages/backend/apps/api/src/route-controllers/program-year/_tests_/e2e/program-year.students.controller.getProgramYears.e2e-spec.ts

```diff
@@ -0,0 +1,157 @@
+import { HttpStatus, INestApplication } from "@nestjs/common";
+import { ProgramYear } from "@sims/sims-db";
+import * as request from "supertest";
+import { DataSource } from "typeorm";
+
+import {
+  BEARER_AUTH_TYPE,
+  createTestingAppModule,
+  getStudentToken,
+  FakeStudentUsersTypes,
+} from "../../../../testHelpers";
+
+describe("ProgramYearStudentsController(e2e)-getProgramYears", () => {
+  let app: INestApplication;
+  let appDataSource: DataSource;
+  beforeAll(async () => {
+    const { nestApplication, dataSource } = await createTestingAppModule();
+    appDataSource = dataSource;
+    app = nestApplication;
+  });
+
+  it("Should get the list of the available program years.", async () => {
+    // Arrange
+    const endpoint = "/students/program-year/options-list";
```

Comment:
Same for https://github.com/bcgov/SIMS/pull/1803/files#r1137781213

Reply:
What should I test in this test if "ignore" the records that are persisted by the DB seeding? Just check if it contains the records I added?

Reply:
Yes, test if yours belongs to the list or select the active ones from DB and check if the list matches with what you got from the API.
The 3 program years records right now were added at the beginning of the project, they are subject to be changed. I would recommend that the test have control of the data tested.

Reply:
As those program years are subject to be changed, I think they shouldn't be used by other tests either, right?